app/models/member.rb

```diff
@@ -221,17 +221,10 @@ def self.nearest_to(place)
   end
 
   def update_newsletter_subscription
-    if confirmed_at_changed? && newsletter # just signed up
-      newsletter_subscribe
-    elsif confirmed_at # i.e. after member's confirmed their account
-      if newsletter_changed? # edited member settings
-        if newsletter
-          newsletter_subscribe
-        else
-          newsletter_unsubscribe
-        end
-      end
+    if newsletter
```

Comment:
This is shorter, but I find it a lot harder to follow than the previous version. Was the problem that rubocop was complaining about the if-clauses being nested too deeply? We could easily trim one level by changing L226 to `if confirmed_at && newsletter_changed?`.